FirebaseAppCheck/Tests/Fixture/AppAttestResponseSuccess.json

```diff
@@ -0,0 +1,4 @@
+{
+    "challenge": "random_challenge",

```

Comment:
We should use a valid base64 encoded string here, e.g. we can generate something the following Swift code:
```
let str = "random_challenge"
let utf8 = str.data(using: .utf8)
let base64String = utf8?.base64EncodedString() // "cmFuZG9tX2NoYWxsZW5nZQ=="
```

Then after updating the response decoding logic as mentioned in the comment above the tests in FIRAppAttestAPIServiceTests can stay unchanged.